src/ml_gym/frontend/dashboard/src/app/datatypes.ts

```diff
@@ -0,0 +1,135 @@
+// =================== API TYPES ======================
+
+interface BaseMessageType {
+    event_id: number;
+}
+
+interface EventInfoType {
+    event_type: string;
+    creation_ts: number;
+}
+
+
+// Job status
+
+export type JobStatusPayloadType = {
+    job_id: number;
+    job_type: string; //<CALC, TERMINATE>
+    status: string; //<INIT, RUNNING, DONE>,
+    experiment_id: string; // <path_to_model>
+    starting_time: number;
+    finishing_time: number;
+    error?: string;
+    stacktrace?: string;
+    device?: string // "GPU {1, ..., n}" or "CPU"
+}
+
+export type JobStatusInnerType = {
+    payload: JobStatusPayloadType
+} & EventInfoType
+
+
+export type JobStatusType = { data: JobStatusInnerType, } & BaseMessageType
+
+
+// Model status
+
+export type ModelStatusPayloadType = {
+    experiment_id: number;
+    status: string; // <TRAINING, EVALUATING>,,
+    num_epochs: number; //<CALC, TERMINATE>
+    current_epoch: number;
+    splits: Array<string>;
+    current_split: string;
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
  current_split: string;
```